classlib/src/main/java/org/teavm/classlib/java/lang/invoke/TMethodHandleStatics.java

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright 2020 adam.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.classlib.java.lang.invoke;
+
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+class TMethodHandleStatics {
```

Comment:
This class looks like copy of OpenJDK class, so I can't accept since it violates OpenJDK license

Reply:
It's not OpenJDK. I've removed anyway.